frontend/src/components/BattleLog.tsx

```diff
@@ -20,10 +21,22 @@ const BattleLog: React.FunctionComponent<Props> = ({
   enemyUnits,
   damageUnit,
 }: Props) => {
+  const allUnits = [...units, ...enemyUnits];
```

Comment:
Please check if this does not break battle royale while fighting with someone, if he disconnects in the `BATTLE` phase.

Reply:
If the previous version worked this should work too... But I'll double-check just to be sure

Reply:
You were of course right, I changed it so that log entries are not rendered if enemy units are missing

Reply:
I'm not checking it I believe you it works.